src/utils/clack/index.ts

```diff
@@ -1457,21 +1467,23 @@ export async function askForToolConfigPath(
  *       this might require adding a custom message parameter to the function
  */
 export async function showCopyPasteInstructions(
-  filename: string,
-  codeSnippet: string,
-  hint?: string,
+  opts: ShowCopyPasteInstructionsOptions,
```

Comment:
I changed the typing here a bit so that this function als accepts an entirely custom instruction message other than composing a predefined one from the filename and hint params. This was necessary for Angular because the wording wouldn't have fitted the predefined message.